setup.py

```diff
@@ -0,0 +1,56 @@
+import ast
+import os
+import re
+from setuptools import setup
+import sys
+
+import typeshed
+
+
+CURRENT_DIR = os.path.dirname(__file__)
+
+
+def get_long_description():
+    readme_md = os.path.join(CURRENT_DIR, "README.md")
+    with open(readme_md, encoding="utf8") as ld_file:
+        return ld_file.read()
+
+
+assert not typeshed.__version__.endswith(".dirty"), (
```

Comment:
Is this really necessary?

Reply:
Mypy and projects I manage have similar checks, we don't want random dirty changes to land on PyPI without us knowing, right?